### PR TITLE
Added alias_manager module

### PR DESCRIPTION
**Pull request by rbietin**

### CODE REVIEW

**Comment by rbietin** [comment]
Sorry for pulling a bunch of commits into this request. Just squash & rebase on merge

File: modules/alias_manager.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+log = logging.getLogger()
+
+
+def initialize_containers(chat_data):
+    if 'user_aliases' not in chat_data:
+        chat_data['user_aliases'] = {}
+
+
+# Checks if user lis present in user_aliases. If no - adds user's full name as default alias.
```
Comment (1over) [inline modules/alias_manager.py]: typo

Comment (1over) [reply modules/alias_manager.py]: make all comments as Python docstrings

File: modules/alias_manager.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+def add_unknown_user(bot, update, chat_data):
+    initialize_containers(chat_data)
+    user_id = update.message.from_user.id
+    if user_id not in chat_data['user_aliases']:
+        chat_data['user_aliases'][user_id] = update.message.from_user.full_name
+        # Message is suppressed for now. Uncomment once the feature is fully enabled and used by other modules.
+        # update.message.reply_text(
+        #     'Оп, а это шо за пидарок? Я буду называть тебя %s.'
+        #     ' Что бы поменять это напиши "/alias_update %d Пидр-Хуидр"' %
```
Comment (1over) [inline modules/alias_manager.py]: use ''.format() instead of % operator

File: modules/alias_manager.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        return
+
+    chat_data['user_aliases'][target_user_id] = target_user_alias
+    update.message.reply_text('окда')
+
+
+# Builds the message with mapping of user ids to aliases for known users.
+def list_of_known_aliases(chat_data):
+    msg = 'вот кого я знаю:\n'
+    return msg + '\n'.join(list(map(lambda x: '%d: %s' % (x[0], x[1]), chat_data['user_aliases'].items())))
```
Comment (1over) [inline modules/alias_manager.py]: you don't need `list()` here

File: modules/alias_manager.py [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        #     (update.message.from_user.full_name, user_id))
+
+
+# Update alias. Returns an error if one of the following conditions has been met:
+# - Call format doesn't match /alias_update <int: user_id> <new_alias>.
+# - Alias is too long.
+# - Alias is used by someone in the chat.
+def alias_update(bot, update, args, chat_data):
+    if len(args) < 2:
+        args = ['']
```
Comment (1over) [inline modules/alias_manager.py]: why not empty list?

**Pull request closed without merging**